student.rb

```diff
@@ -1,7 +1,11 @@
 require_relative 'person'
+require './class_room'
+require './book'
+require './rental'
 
 class Student < Person
```

Comment:
- [ ] It will be great to have a method that acts as belong_to association between the student and the classroom. For example: 
```
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```
You can learn more about setting the belongs to association [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`)